tests/test_aux_files/patch/bad_recipe_files/missing_colon.yaml

```diff
@@ -0,0 +1,2 @@
+package
+  name: {{ name|lower }}  # [unix]
```

Comment:
Good job on this one. I don't know why I didn't think of this kind of recipe corruption sooner.